### ISSUE TITLE
Export list of failed imports

### ISSUE THREAD

**Issue by bitboxer** [bookwyrm-social/bookwyrm#2979]
**Is your feature request related to a problem? Please describe.**

I have a failed import with 7 pages of items that need fixing. This is a bit overwhelming to have that in bookwyrm.

**Describe the solution you'd like**

I would love to be able to pull it out of bookworm and in an Excel sheet to have a "todo list" and a way to massage the data a bit to make it work again. Ideally, with a way to re-import that exported CSV to try again.

**Describe alternatives you've considered**

I tried to write a small script to export the data from the page, but a csv would be so much nicer.

**Comment by mouse-reeve** [comment]
Good idea! I believe we preserve the rows in the database, so it should be possible to export those rows using the same logic as the data exporter without having to do anything too tricky with the data
